### PR TITLE
Add Desktop Notification Options

### LINKED ISSUE #77: Add Desktop Notifications as an option

### PR DESCRIPTION
**Pull request by dennypradipta**
# MonikaConfigGenerator PR

## What does this PR solve?

This PR fixes #77 

## How does this PR solve the problem?

This PR adds an option for Desktop notifications

## How to test this PR?

Try to create a config using all options available in the wizard, and add desktop notifications as the notification channel

### CODE REVIEW

**Review by nicnocquee — APPROVED** [review]

File: components/notif-card/index.tsx [hunk 1]
```diff
@@ -73,6 +73,26 @@ const NotifCard: FunctionComponent<NotifCardProps> = ({ id, type }) => {
                   </div>
                 );
               })}
+            {type === 'desktop' && (
+              <>
+                <p>There are prerequisites for using Desktop notifications:</p>
```
Comment (raosan) [inline components/notif-card/index.tsx:78]: i think it's singular: notification without `s`

File: pages/notifications.tsx [hunk 2]
```diff
@@ -594,6 +605,27 @@ function SendgridForm({ formHelper }: ChannelFormProps): JSX.Element {
   );
 }
 
+function DesktopForm(): JSX.Element {
+  return (
+    <>
+      <p>There are prerequisites for using Desktop notifications:</p>
```
Comment (raosan) [inline pages/notifications.tsx:611]: here too

**Review by raosan — APPROVED** [review]
approved with little suggestions

File: components/notif-card/index.tsx (hunk 1 above)
Comment (dennypradipta) [reply components/notif-card/index.tsx]: Thanks man!

File: pages/notifications.tsx (hunk 2 above)
Comment (dennypradipta) [reply pages/notifications.tsx]: Thanks man!

**Pull request merged**